src/typescript/decorators.d.ts

```diff
@@ -0,0 +1,54 @@
+/// <reference path="../../node_modules/reflect-metadata/Reflect.d.ts" />
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+/**
+ * A TypeScript class decorator that defines a custom element with name
+ * `tagname` and the decorated class.
+ */
+export declare function customElement(tagname: string): (clazz: any) => void;
+export interface PropertyOptions {
+    notify?: boolean;
+}
+/**
+ * A TypeScript property decorator factory that defines this as a Polymer
+ * property.
+ *
+ * This function must be invoked to return a decorator.
```

Comment:
Here and elsewhere, I think giving a good and a bad example of use would be better documentation than this caveat. It strikes me as only making sense to people who already understand decorators well enough to not need the documentation. :)